js/defs.bzl

```diff
@@ -26,15 +26,29 @@ load(
     _js_image_layer = "js_image_layer",
 )
 
+# buildifier: disable=function-docstring
 def js_binary(**kwargs):
+    include_npm_deps = kwargs.pop("include_npm_deps", False)
+
+    # For backward compat
+    # TODO(3.0): remove backward compat handling
+    include_npm_linked_packages = kwargs.pop("include_npm_linked_packages", None)
```

Comment:
Why not drop it in v2? Maybe add this warning in v1? I don't think we we necessarily *need* a full semver warning people?

Reply:
Adding the warning in V1 means that if someone updates from an older version of v1 to v2 then they will get a fail instead of a WARNING. Keeping backward compat with a warning is generally good practice where possible. Was not possible for all of the breaking changes in V2 but for this one we can give users a nice hint of what field to upgrade to instead of getting the Bazel failure of "invalid attribute".